geoutils/raster/distributed_computing/multiproc.py

```diff
@@ -317,10 +333,122 @@ def map_multiproc_collect(
         for results in tasks:
             result, dst_tile = config.cluster.get_res(results)
             if return_tile:
-                list_results.append((result, return_tile))
+                list_results.append((result, dst_tile))
             else:
                 list_results.append(result)
         return list_results
 
     except Exception as e:
         raise RuntimeError(f"Error retrieving terrain attribute from multiprocessing tasks: {e}")
+
+
+def _multiproc_reproject_per_block(
```

Comment:
Do we also need a wrapper here, isn't it the exact same call as `_reproject_per_block` directly?